### DISCUSSION TITLE
Detect and Recording toggle buttons turns off after restart

### DISCUSSION THREAD

**Discussion by hajen5** [blakeblackshear/frigate#10561] [General Support]
After restart of Homeassistant detect and recording turns off

Frigate 0.13.2
Core 2024.1.5
Supervisor 2024.01.1
Operating System 11.4
Frontend 20240104.0
Google Coral
HW RPi4 -64

<img width="257" alt="Skärmavbild 2024-02-20 kl  10 51 05" src="https://github.com/blakeblackshear/frigate/assets/80976150/8d9c870b-e98d-410a-9863-a8721843de07">

**Comment by InAmberClad** [comment]
I've been having this same issue for both the detect and record toggles for the last few weeks. Quite frustrating.

I checked using MQTT Explorer and found both topics were flagged as retained for some reason.

Frigate 0.13.2 running in Docker (Proxmox LXC) with a Google Coral PCIE passed through.

**Comment by NickM-27** [comment]
if you see a retained topic then it should be removed

**Reply by InAmberClad** [reply]
Sorry I didn't make it clear in my post but I did remove the retained flag but found it came back after some time and the issue repeated.

**Reply by InAmberClad** [reply]
I've just checked again and the retained flags have returned since I deleted them 3 days ago.

**Reply by NickM-27** [reply]
On what topic?

**Reply by InAmberClad** [reply]
The detect and record topics. It only happens for one of my cameras, the only difference is this camera has audio detection enabled.

**Reply by NickM-27** [reply]
there are multiple topics for record and detect

**Reply by InAmberClad** [reply]
Back at a computer now so I can give a detailed response.

frigate/lounge/detect/state
frigate/lounge/recordings/state
frigate/lounge/recordings/set

All of the above are retained topics that I removed the retained flag from a few days ago.

**Reply by NickM-27** [reply]
that likely means you're either running an old version of frigate or an old version of the integration, since neither of the latest version sets either as retained

**Reply by InAmberClad** [reply]
Frigate 0.13.2
Frigate Integration 5.0.1

I will update the integration and see if the issue persists.

**Reply by InAmberClad** [reply]
The issue is still happening.
